Jenkinsfile

```diff
@@ -8,10 +8,10 @@ podTemplate(
         envVars: [
           envVar(key: 'CI', value: 'true'),
         ],
-        resourceRequestCpu: '4000m',
-        resourceRequestMemory: '4500Mi',
-        resourceLimitCpu: '4000m',
-        resourceLimitMemory: '4500Mi',
+        resourceRequestCpu: '8000m',
```

Comment:
the nodes we use are n1-standard-8. so there are 8 cpus in total and only 7.something will be available for use. try something less than 7.5